lib/utils/resolveObject.js

```diff
@@ -62,36 +66,29 @@ function traverseObj(obj) {
         traverseObj( obj[key] );
       } else {
         if (typeof obj[key] === 'string' && obj[key].indexOf('{') > -1) {
-          try {
-            obj[key] = compile_value( obj[key], [key] );
-          }
-          catch (e) {
-            errors.push(e);
```

Comment:
Because we are removing this, we should remove the instantiation of the errors variable above.